Documentation/helm-values.rst

```diff
@@ -2050,8 +2050,8 @@
      - ``{"digest":"","override":null,"pullPolicy":"Always","repository":"quay.io/cilium/cilium-ci","tag":"latest","useDigest":false}``
    * - :spelling:ignore:`imagePullSecrets`
      - Configure image pull secrets for pulling container images
-     - string
-     - ``nil``
+     - list
+     - ``[]``
```

Comment:
Is this an incompatible change? Users may have configured this setting already with a direct string, so changing the type like this could break their existing configuration..?

Reply:
Maybe if we think this is the right type for this variable moving forward, we can just document this in the upgrade notes `Documentation/operations/upgrade.rst` that the setting is changing the value type. (Same for any others that we consciously change)

Reply:
Not really sure if this would work at all. I suspect helm erroring if people actually specify a string here.

Reply:
Huh, I see what you mean, the API reference specifies this an an array. I'm confused how we ended up with this broken option in the Helm charts in the first place. Well, thanks for the fix :)

Reply:
> I suspect helm erroring if people actually specify a string here.

Even before this PR